tests/chunk.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
What's the purpose of this file? The location of this file is a bit out-of-pattern from the rest of the Utopia libraries.

Reply:
Wrongfully committed 😄